src/RichText.js

```diff
@@ -0,0 +1,82 @@
+import {Editor, ContentState, convertFromHTML, DefaultDraftBlockRenderMap, DefaultDraftInlineStyle, CompositeDecorator} from 'draft-js';
+const ReactDOM = require('react-dom');
+var sdk = require('./index');
+
+const BLOCK_RENDER_MAP = DefaultDraftBlockRenderMap.set('unstyled', {
+    element: 'p' // draft uses <div> by default which we don't really like
+});
+
+const styles = {
+    BOLD: 'strong',
+    CODE: 'code',
+    ITALIC: 'em',
+    STRIKETHROUGH: 's',
+    UNDERLINE: 'u'
+};
+
+export function contentStateToHTML(contentState:ContentState): String {
+    const elem = contentState.getBlockMap().map((block) => {
+        const elem = BLOCK_RENDER_MAP.get(block.getType()).element;
+        const content = [];
```

Comment:
Whilst I appreciate it is basically `let` with an additional contract  that you can't reassign the variable, I don't like how you're basically using `const` as a `let` replacement. I don't like it for these reasons:
 - It obfuscates cases where having a variable be `const` is important (since by default *all* your variables are now `const`). 
 - It's confusing for people coming from other languages, which is particularly important when context-switching back into Javascript, because it's a very very sloppy `const` which isn't actually protecting the underlying data type (e.g. properties)
 - Re-assigning to a `const` variable would *silently fail* for a number of browsers for a time, this is still the case on a lot of stable versions of Node (e.g. 4). This would be nightmarish to debug as the code would not be raising any errors at all!

The oft-mentioned counter point is that it is basically a safer `let` is valid, but that only protects you from redeclaring that variable in that scope. This doesn't tend to be a common failure mode since your functions shouldn't be so long that you can't hold all the variable names of a function in your head (and if it is, `const` will not be a panacea).

This is a personal thing though, and may not reflect @ara4n, @dbkr and @richvdh 's feelings on the matter.